pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -236,31 +254,43 @@ func (c *Cluster) makeDeployment(name, daemonName string) *extensions.Deployment
 	return d
 }
 
-func (c *Cluster) mgrContainer(name, daemonName string) v1.Container {
-
+func (c *Cluster) mgrContainer(name, daemonName, clusterName, confFilePath, keyringPath string) v1.Container {
 	return v1.Container{
+		Command: []string{
```

Comment:
Note that the osd pod with the init container is generated [here](https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/osd/pod.go#L213). There may be similar patterns to re-use or else update there as we find better patterns.

Reply:
I like the way the OSD looks with a description of the containers, and the env vars and volumes are gotten from methods. I might change `command` in the main container to be `daemonCommand` or some such thing to be more explicit.